### DISCUSSION TITLE
Max number of parallel conversions via Libreofice

### DISCUSSION THREAD

**Discussion by andretti1977** [gotenberg/gotenberg#348] [Q&A] [answered]
Is there a way to specify how many concurrent conversions may happen? I.E. if we send 100 conversion requests, Goteborg tries to process them concurrently, but is it possible to set max X conversion concurrently?

**Comment by gulien** [comment, marked as answer]
Currently, it's not possible to set a maximum number of concurrent instances.

I'm currently working on a "metrics" system (see #323). One of the metrics is the number of LibreOffice instances, and it should be possible to create a custom module that provides a middleware with the following behavior: there are X LibreOffice instances, X > to max number of instances, let's return a `503` response.

**Reply by andretti1977** [reply]
Ok thank you very much, it would be extremely interesting if when introducing metrics, you could add the middleware "limiter".
